packages/sdk/src/accountTransactions.ts

```diff
@@ -108,7 +108,10 @@ export class SimpleTransferHandler
     }
 
     deserialize(serializedPayload: Cursor): SimpleTransferPayload {
-        const toAddress = AccountAddress.fromBuffer(Buffer.from(serializedPayload.read(32)));
+        // 1. Correct the INPUT type conflict using the triple assertion
+        const inputBuffer = Buffer.from(serializedPayload.read(32)).buffer as any as ArrayBuffer;
+
+        const toAddress = AccountAddress.fromBuffer(inputBuffer);
```

Comment:
I'm a hesitant to approve of this without knowing why this happens, i.e. why has it been possible up to this point and now all of a sudden we need to do force casts to make it compile 🤔

I think we need to look into whether the `buffer` dependency version changed and why that would be incompatible with the current signature. We generally use `ArrayBuffer` as the type to represent memory buffers (usually a byte array) for input to functions everywhere. It would not be nice to have to cast like this for all uses of the functions in question as that makes us lose compile time errors for all of this.